viewer/src/views/threejs/cad/materials.ts

```diff
@@ -39,6 +40,9 @@ export function createMaterials(treeIndexCount: number): Materials {
   visibility.fill(255);
   const overrideColorPerTreeIndex = new THREE.DataTexture(colors, textureDims.width, textureDims.height);
   const overrideVisibilityPerTreeIndex = new THREE.DataTexture(visibility, textureDims.width, textureDims.height);
+  
+  var matCapTexture = new THREE.Texture(matCapTextureImage);
+  matCapTexture.needsUpdate = true;
```

Comment:
is setting needsUpdate necessary for new textures?

Reply:
Well, everything just turns black if I don't have this. So I assume you need it.